cluster/addons/istio/auth/istio-auth.yaml

```diff
@@ -593,6 +603,25 @@ spec:
   version: v1alpha2
 ---
 
+kind: CustomResourceDefinition
+apiVersion: apiextensions.k8s.io/v1beta1
+metadata:
+  name: solarwindses.config.istio.io
+  labels:
+    k8s-app: istio
+    addonmanager.kubernetes.io/mode: Reconcile
+    package: solarwinds
+    istio: mixer-adapter
+spec:
+  group: config.istio.io
+  names:
+    kind: solarwinds
+    plural: solarwindses
```

Comment:
This is weird plural :).

Reply:
I think it's middle earth english - one hobbit, two hobbitses.